### PR TITLE
Refactors occurrences of admin:admin and remove integtest.sh

### PR DESCRIPTION
**Pull request by DarshitChanpura**
### Description

There were recent changes to security demo configuration setup which now requires a custom admin Password to be setup. This PR changes workflows that previously used admin as password.

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by DarshitChanpura** [comment]
This change should be backported to 2.x

**Comment by derek-ho** [comment]
@SuZhou-Joe @xluo-aws can you take a look at this PR and let us know what you think? Full context is that now install demo configuration requires a strong password to be set, instead of just default to admin:admin.

File: .github/workflows/release-e2e-workflow-template-windows.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ jobs:
       TERM: xterm
       # make Node run in ipv4 first so that cypress can detect 5601 port in CI environment
       NODE_OPTIONS: '--max-old-space-size=6144 --dns-result-order=ipv4first'
+      # 2.12 onwards security demo configuration require a custom admin password
+      OPENSEARCH_INTIAL_ADMIN_PASSWORD: 'myStrongPassword123!'
```
Comment (Hailong-am) [inline .github/workflows/release-e2e-workflow-template-windows.yml:26]: For curious, is this the new default password of admin user? or security plugin will honer env variable `OPENSEARCH_INTIAL_ADMIN_PASSWORD`

File: .github/workflows/release-e2e-workflow-template-windows.yml [hunk 2]
```diff
@@ -104,8 +106,8 @@ jobs:
           else
               echo "Keep Dashboards Security"
               bin/opensearch-dashboards.bat &
-              timeout 300 bash -c 'while [[ "$(curl -k http://localhost:5601/api/status -u admin:admin | jq -r '.status.overall.state')" != "green" ]]; do echo sleeping 5; sleep 5; done'
-              curl -sk localhost:5601/api/status -u admin:admin | jq
+              timeout 300 bash -c 'while [[ "$(curl -k http://localhost:5601/api/status -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} | jq -r '.status.overall.state')" != "green" ]]; do echo sleeping 5; sleep 5; done'
```
Comment (SuZhou-Joe) [inline .github/workflows/release-e2e-workflow-template-windows.yml:109]:
```suggestion
              timeout 300 bash -c 'while [[ "$(curl -k http://localhost:5601/api/status -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} | jq -r '.status.overall.state')" != "green" ]]; do echo sleeping 5; sleep 5; done'
```

File: .github/workflows/release-e2e-workflow-template-windows.yml [hunk 3]
```diff
@@ -104,8 +106,8 @@ jobs:
...
+              curl -sk localhost:5601/api/status -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} | jq
```
Comment (SuZhou-Joe) [inline .github/workflows/release-e2e-workflow-template-windows.yml:110]:
```suggestion
              curl -sk localhost:5601/api/status -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} | jq
```

File: .github/workflows/release-e2e-workflow-template.yml [hunk 4]
```diff
@@ -61,8 +63,8 @@ jobs:
           else
               echo "Keep OpenSearch Security"
               ./opensearch-tar-install.sh &
-              timeout 900 bash -c 'while [[ "$(curl -o /dev/null -w ''%{http_code}'' -u admin:admin -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
-              curl https://localhost:9200 -u admin:admin --insecure
+              timeout 900 bash -c 'while [[ "$(curl -o /dev/null -w ''%{http_code}'' -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```
Comment (SuZhou-Joe) [inline .github/workflows/release-e2e-workflow-template.yml:66]:
```suggestion
              timeout 900 bash -c 'while [[ "$(curl -o /dev/null -w ''%{http_code}'' -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```

File: .github/workflows/release-e2e-workflow-template.yml [hunk 5]
```diff
@@ -61,8 +63,8 @@ jobs:
...
+              curl https://localhost:9200 -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} --insecure
```
Comment (SuZhou-Joe) [inline .github/workflows/release-e2e-workflow-template.yml:67]:
```suggestion
              curl https://localhost:9200 -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} --insecure
```

File: .github/workflows/release-e2e-workflow-template.yml [hunk 6]
```diff
@@ -92,8 +94,8 @@ jobs:
           else
               echo "Keep Dashboards Security"
               bin/opensearch-dashboards serve ${{ inputs.osd-serve-args }} &
-              timeout 300 bash -c 'while [[ "$(curl -u admin:admin -k http://localhost:5601/api/status | jq -r '.status.overall.state')" != "green" ]]; do sleep 5; done'
-              curl http://localhost:5601/api/status -u admin:admin --insecure
+              timeout 300 bash -c 'while [[ "$(curl -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k http://localhost:5601/api/status | jq -r '.status.overall.state')" != "green" ]]; do sleep 5; done'
+              curl http://localhost:5601/api/status -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} --insecure
```
Comment (SuZhou-Joe) [inline .github/workflows/release-e2e-workflow-template.yml:98]:
```suggestion
              timeout 300 bash -c 'while [[ "$(curl -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k http://localhost:5601/api/status | jq -r '.status.overall.state')" != "green" ]]; do sleep 5; done'
              curl http://localhost:5601/api/status -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} --insecure
```

File: .github/workflows/release-signoff-chrome.yml [hunk 7]
```diff
@@ -29,7 +31,7 @@ jobs:
           tar -xzf opensearch-${{ env.VERSION }}-linux-x64.tar.gz
           cd opensearch-${{ env.VERSION }}/
           ./opensearch-tar-install.sh &
-          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:admin -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
+          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```
Comment (SuZhou-Joe) [inline .github/workflows/release-signoff-chrome.yml:34]:
```suggestion
          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```

File: .github/workflows/release-signoff-chromium-ad-only.yml [hunk 8]
```diff
@@ -29,7 +31,7 @@ jobs:
           tar -xzf opensearch-${{ env.VERSION }}-linux-x64.tar.gz
           cd opensearch-${{ env.VERSION }}/
           ./opensearch-tar-install.sh &
-          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:admin -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
+          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```
Comment (SuZhou-Joe) [inline .github/workflows/release-signoff-chromium-ad-only.yml:34]:
```suggestion
          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:${{ env.OPENSEARCH_INTIAL_ADMIN_PASSWORD }} -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```

File: .github/workflows/release-e2e-workflow-template-windows.yml (hunk 1 above)
Comment (SuZhou-Joe) [reply .github/workflows/release-e2e-workflow-template-windows.yml:26]: @derek-ho Same question here, could you clarify this?

Comment (Hailong-am) [reply .github/workflows/release-e2e-workflow-template-windows.yml:26]:
```suggestion
      OPENSEARCH_INITIAL_ADMIN_PASSWORD: 'myStrongPassword123!'
```

**Review by Hailong-am — CHANGES_REQUESTED** [review]
please update all reference accordingly.

File: .github/workflows/release-e2e-workflow-template-windows.yml (hunk 1 above)
Comment (derek-ho) [reply .github/workflows/release-e2e-workflow-template-windows.yml:26]: Thanks for the catch! And yes, when running the install demo config we will honor whatever you put as that variable.

**Comment by derek-ho** [comment]
Closing this in favor of: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/981/files

**Comment by DarshitChanpura** [comment]
Closing in favor of: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/981/files

**Pull request closed without merging**